src/java_bytecode/generate_java_generic_type.cpp

```diff
@@ -1,343 +0,0 @@
-/*******************************************************************\
-
- Module: Generate Java Generic Type - Instantiate a generic class with
-         concrete type information.
-
- Author: DiffBlue Limited. All rights reserved.
-
-\*******************************************************************/
-
-#include <iterator>
-
-#include "generate_java_generic_type.h"
-#include "generic_arguments_name_builder.h"
-#include <util/namespace.h>
-#include <java_bytecode/java_types.h>
-#include <java_bytecode/java_utils.h>
-
-generate_java_generic_typet::generate_java_generic_typet(
-  message_handlert &message_handler):
-    message_handler(message_handler)
-{}
-
-/// Small auxiliary function, to print a single generic argument name.
-/// \param argument argument type
-/// \return printed name of the argument
-static std::string argument_name_printer(const reference_typet &argument)
```

Comment:
Probably worth taking a note of this commit somewhere, as some of the utilities seem like they will be useful?

Reply:
I agree, I made a note about this function in my notes. I also added a comment in TG-1278 and TG-1871 which seem to be the most relevant open tickets for pretty printing generic types.